src/condor_utils/file_transfer.cpp

```diff
@@ -4718,6 +4726,47 @@ MyString FileTransfer::GetSupportedMethods() {
 	return method_list;
 }
 
+int FileTransfer::InitializeJobPlugins(const ClassAd &job, CondorError &e, StringList &infiles)
+{
+	if ( ! I_support_filetransfer_plugins || ! plugin_table) {
+		return 0;
+	}
+
+	std::string job_plugins;
+	if ( ! job.LookupString(ATTR_TRANSFER_PLUGINS, job_plugins)) {
+		return 0;
+	}
+
+	StringTokenIterator plugins(job_plugins, 100, ";");
```

Comment:
Could you add a comment about what this is doing?  I.e., in the ticket, you specify the format:

> This would add the following to the job.
>    ```TransferPlugins = "ZKM:/path/to/zkm_plug.exe; HTTP,HTTPS:my_http_plugin.exe"```

Would be good to have it above the block as it's a bit difficult to follow string parsing in C++.